From 28033281b6e774dbb240d8f54cd5b6f5e22351a7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marcus=20M=C3=BCller?= <marcus@hostalia.de>
Date: Mon, 31 Mar 2014 17:29:54 +0200
Subject: [PATCH 12/13] made the usage of mt19937 Boost-pre-1.47 compatible

"correct" namespace is boost::random::, but mt19937 still exists in boost::, so
fixing this was just using the same name like in the other files that use
mt19937
---
 gr-blocks/lib/message_strobe_random_impl.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gr-blocks/lib/message_strobe_random_impl.h b/gr-blocks/lib/message_strobe_random_impl.h
index 277c113..32eddb5 100644
--- a/gr-blocks/lib/message_strobe_random_impl.h
+++ b/gr-blocks/lib/message_strobe_random_impl.h
@@ -45,7 +45,7 @@ namespace gr {
       void run();
       float next_delay();
 
-      boost::random::mt19937 d_rng;
+      boost::mt19937 d_rng;
       boost::shared_ptr< boost::variate_generator <boost::mt19937, boost::poisson_distribution<> > > d_variate_poisson;
       boost::shared_ptr< boost::variate_generator <boost::mt19937, boost::normal_distribution<> > > d_variate_normal;
       boost::shared_ptr< boost::variate_generator <boost::mt19937, boost::uniform_real<> > > d_variate_uniform;
-- 
1.7.10.4

